core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -234,11 +234,6 @@ public static String getAppVersion() {
         public Options() {
         }
 
-        public Options(@NonNull final String configFileName, @NonNull final HttpServiceModule httpServiceModule) {
```

Comment:
@danielpassos I wasn't sure of removing this but I can see that Options is meant to be used like fluent api
```java
new Options()
    .setConfigFileName()
    .setFoo()
```
Plus is not being used used anywhere. WDYT?

Reply:
If it's not used... 

![Kill it with fire](http://i0.kym-cdn.com/photos/images/original/000/165/945/444.gif)